addons/mail/static/src/components/welcome_view/welcome_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.WelcomeView" owl="1">
+        <div class="o_WelcomeView h-100 d-flex flex-column justify-content-center align-items-center bg-light">
+            <h1 class="font-weight-light">You've been invited to a meeting!</h1>
+            <h2 class="m-5" t-esc="messaging.companyName"/>
+            <div class=" d-flex justify-content-center">
+                <MediaPreview class="mr-5" localId="props.mediaPreviewLocalId"/>
+                <div class="d-flex flex-column justify-content-center">
```

Comment:
This div needs to be able to move above (or below) the preview in mobile, not stay stuck on the right.